frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmListModel.java

```diff
@@ -788,19 +788,19 @@ private void edit() {
 
     }
 
-    private void confirmChangedBiosType() {
-        if (getSelectedItem() == null || ((UnitVmModel) getWindow()).getIsNew()) {
+    private void confirmBiosTypeUpdate(){
+        VM vm = getSelectedItem();
+        UnitVmModel model = (UnitVmModel) getWindow();
+        if (getSelectedItem() == null || ((UnitVmModel)getWindow()).getIsNew()) {
```

Comment:
`getSelectedItem() == null` --> `vm == null`
That was the reason why I suggested to move up the initialization of `vm` :)

Reply:
The same for `model`.
`((UnitVmModel)getWindow()).getIsNew()` can be simplified to `model.getIsNew()`.

Reply:
Thanks for fixing the issues, no more comments :)